static/js/pv_tree.tsx

```diff
@@ -0,0 +1,163 @@
+import React, {Component} from "react";
+import hierarchy from "../../tools/pv_tree_generator/hierarchy_golden.json";
+//import hierarchy from "../data/hierarchy.json";
+
+// Fields of each node from json file
+interface NodeType {
+  show: string; 
+  title: string; 
+  selected: string;
```

Comment:
selected, expanded, placeTypes can be removed?

Reply:
We actually don't need this interface. Removed it.